less/navigation.less

```diff
@@ -1 +1,41 @@
-// Navigation Styles here
\ No newline at end of file
+// Navigation Styles here
+.nav-bar {
+    max-width: 1000px;
```

Comment:
I'd put a max-width of 800 here since thats the max-width of the rest of the page